themes/jb/assets/css/main.sass

```
@@ -7,3 +7,10 @@
 @import "helpers/_all"
 #wrapper
   padding-top: 4rem
+
+.host-list .columns
+  @include until(600px)
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    .column:first-child:last-child
+      grid-column: 1 / span 2;
```

SASS does not use semi colons on the ends of the lines. While I don't think it breaks anything, they should be removed to match the rest of the stylesheets
```suggestion
    display: grid
    grid-template-columns: repeat(2, 1fr)
    .column:first-child:last-child
      grid-column: 1 / span 2
```